redist/org.xiaoyifang.GoldenDict_NG.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Vitaly Zaitsev <vitaly@easycoding.org> -->
+<component type="desktop">
+  <id>org.xiaoyifang.GoldenDict_NG</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <name>GoldenDict-NG</name>
+  <summary>Dictionary lookup program</summary>
+  <description>
+    <p>
+        GoldenDict-NG is a feature-rich dictionary lookup program, supporting multiple
+        dictionary formats, featuring perfect article rendering with the complete
+        markup, illustrations and other content retained, and allowing you to type
+        in words without any accents or correct case.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://xiaoyifang.github.io/goldendict-ng/img/linux_genshin.webp</image>
+    </screenshot>
+    <screenshot>
+      <image>https://xiaoyifang.github.io/goldendict-ng/img/mac_black.webp</image>
+    </screenshot>
+    <screenshot>
+      <image>https://xiaoyifang.github.io/goldendict-ng/img/windows_white.webp</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://github.com/xiaoyifang/goldendict-ng</url>
+  <update_contact>https://github.com/xiaoyifang/goldendict-ng</update_contact>
+  <launchable type="desktop-id">org.xiaoyifang.GoldenDict_NG.desktop</launchable>
+  <content_rating type="oars-1.0" />
+  <provides>
+    <id>org.goldendict_ng.desktop</id>
```

Comment:
should this id also be changed ?

Reply:
The original `privdes->id` is just this:
![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/61fffc49-2ef6-4c92-a003-ec69e013fbce)

---

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-provides

This id is used for other software that uses `<require>` tag to describe that they depend on another software that `<provides>`. Others can also `<suggest>` an id that we `<provides>`

Just append a _ng to the original should be OK.

(However, I failed to find who actually uses the `<require>` or `<suggest>` tag :sweat_smile: )

---

Most GUI apps don't even have `provides -> id` (like qbittorrent, okular....). 

https://appstream.ubuntu.com/focal/universe/metainfo/index.html

Reply:
ok